### PR TITLE
Fixed c356824

### PR DESCRIPTION
**Pull request by TetianaParanich**
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/599/parameters/

### CODE REVIEW

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 1]
```diff
@@ -125,9 +125,9 @@ const checkInstanceNotes = (noteType, noteContent) => {
 };
 
 const waitInstanceRecordViewOpened = (title) => {
-  // need to wait untill updated instance will be displayed
-  cy.wait(1500);
   cy.expect(Pane({ id:'pane-instancedetails' }).exists());
+  // need to wait untill updated instance will be displayed
```
Comment (penfriend) [inline cypress/support/fragments/inventory/inventoryInstance.js:129]: untill => until

File: cypress/e2e/data-import/importing-marc-bib-files/single-record-import-with-running-large-job.cy.js [hunk 2]
```diff
@@ -71,9 +71,9 @@ describe('ui-data-import: Inventory single record import is not delayed when lar
       Logs.checkFileIsRunning(fileName);
```
Comment (penfriend) [inline cypress/e2e/data-import/importing-marc-bib-files/single-record-import-with-running-large-job.cy.js:71]: 'getLongDelay' is defined but never used. Allowed unused vars must match /React/u

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/inventoryInstance.js]: Done

File: cypress/e2e/data-import/importing-marc-bib-files/single-record-import-with-running-large-job.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/importing-marc-bib-files/single-record-import-with-running-large-job.cy.js:70]: I deleted it

**Review by penfriend — APPROVED** [review]

File: cypress/support/fragments/data_import/match_profiles/matchProfileEdit.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default {
     cy.do(Button({ id:'criterion-value-type' }).click());
     cy.expect(criterionValueTypeSelectionList.exists());
     cy.do(criterionValueTypeSelectionList.find(SelectionOption('Admin data: Instance UUID')).click());
-    // need to wait untill value weill be selected
+    // need to wait until value weill be selected
```
Comment (penfriend) [inline cypress/support/fragments/data_import/match_profiles/matchProfileEdit.js:23]: weill => will

Comment (TetianaParanich) [reply cypress/support/fragments/data_import/match_profiles/matchProfileEdit.js]: Done

**Pull request merged**